lxd/device/config/device_runconfig.go

```diff
@@ -33,6 +33,12 @@ type RootFSEntryItem struct {
 	Opts []string // Describes the mount options associated with the filesystem.
 }
 
+// USBDeviceItem represents a single USB device matched from LXD USB device specification.
+type USBDeviceItem struct {
+	DerivedDeviceName string
```

Comment:
Can we just call this `DeviceName` please as these structs represent actual USB devices, and thus it is the actual USB device name for the instance driver to use in its config. The fact its been derived from the logical LXD device name isn't something the user of this struct needs to know/care about.

Reply:
Sure!

Reply:
Done